core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -532,33 +532,35 @@ private SslData createSslData(
   private Metadata fetchMetadata() {
     try {
       ConnectSettings instanceMetadata =
-          apiClient.connect().get(projectId, instanceId).execute();
+          apiClient.connect().get(
```

Comment:
Let's match the Google Style guide and only break lines if they're over 100 characters.